networks/src/lib.rs

```diff
@@ -197,14 +197,16 @@ pub fn default_network_version() -> NetworkVersion {
 }
 
 pub mod de_network_version {
+    use std::borrow::Cow;
+
     use forest_shim::version::NetworkVersion;
     use serde::{de, Deserialize, Deserializer, Serialize, Serializer};
 
     pub fn deserialize<'de, D>(deserializer: D) -> Result<NetworkVersion, D::Error>
     where
         D: Deserializer<'de>,
     {
-        let version: &str = Deserialize::deserialize(deserializer)?;
+        let version: Cow<str> = Deserialize::deserialize(deserializer)?;
```

Comment:
An interesting case of https://github.com/serde-rs/serde/issues/1413#issuecomment-494892266 surfaced. Hopefully, the unit tests covered all cases where it would fail.